Probesys/bileto
# fix: Fix visibility of the image of confidential messages

Related to https://github.com/Probesys/bileto/issues/154

Changes proposed in this pull request:

- Add a user-secret-light icon
- Use this icon in dark mode as background of confidential messages

How to test the feature manually:

1. Select dark mode in your preferences
2. Open a ticket
3. Add a confidential message
4. Check the image is visible in the background of the message

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
